src/Confluent.Kafka/Admin/CreatePartitionsException.cs

```diff
@@ -34,12 +34,12 @@ public class CreatePartitionsException : Exception
         ///     (whether or not they were in error). At least one of these
         ///     results will be in error.
         /// </param>
-        public CreatePartitionsException(List<CreatePartitionsExceptionResult> results)
-            : base(
+        public CreatePartitionsException(List<CreatePartitionsReport> results)
+            : base(new Error(ErrorCode.Unknown,
```

Comment:
same comments as `AlterConfigs` case re: deriving from `KafkaException` and the `CreatePartitionsReport` class name change.